lib/rucio/web/rest/flaskapi/v1/main.py

```diff
@@ -72,14 +72,17 @@ def apply_endpoints(app, modules):
             raise ConfigurationError(f'"{blueprint_module}" from the endpoints configuration value did not have a blueprint')
 
 
-try:
-    endpoints = config_get('api', 'endpoints', raise_exception=False, default='')
-    endpoints = list(filter(bool, map(str.strip, endpoints.split(sep=','))))
-except RuntimeError:
-    endpoints = None
+endpoints = config_get_list('api', 'endpoints', raise_exception=False, default=DEFAULT_ENDPOINTS)
 
 if not endpoints:
```

Comment:
I think there may be a clash between what I had in my mind and what you’re trying to achieve.

I think this conditional no longer makes sense. Suppose the following configuration:
```ini
[api]
endpoints_add = foo
```
It will be rendered ineffective. So when I suggested replacing `default=[]` with `default=DEFAULT_ENDPOINTS`, I expected this bit to have been removed.

The only question is how should a configuration like:
```ini
[api]
endpoints = foo,bar
endpoints_add = baz
```
be interpreted. In my opinion, I don’t see a need to actively prevent it. Resulting with `endpoints = ['foo', 'bar', 'baz']` seems acceptable.

Did you intend to have `api.endpoints_add` and `api.endpoints_removed` be silently ignored when `api.endpoints` was set?

Reply:
Yes, the original idea was to silently ignore them when `endpoints` is explicitly set after some discussion with @bari12 , maybe he can comment here. I don't mind that they are not ignored after all.

I added an exception when the same point is used in `add` and `remove` to prevent difficult to predict behaviour (will it be added or removed first? etc.). Do you agree with this?

Reply:
I don’t have any say in this, nor a strong opinion. But if that was your intent, then please ignore my previous comment about setting `default=DEFAULT_ENDPOINTS`.

Reply:
In any case I think silently ignoring them is a bad idea, I will throw saying that you either use `endpoints` explicitly or add to defaults.